src/components/notched-outline/examples/notched-outline-basic.tsx

```
@@ -0,0 +1,154 @@
+
+    @State()
+    private hasFloatingLabel = false;
+
+    @State()
+    private inputValue: string;
+
+    public render() {
+        const id = 'abcd';
+
+        return [
+            <section>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)